docs/use/ble.md

```diff
@@ -152,6 +152,55 @@ you can also accept all the devices by the following command:
 
 The default value is set into config_BT.h
 
+## Read/write BLE characteristics over MQTT (ESP32 only)
+
+The gateway can read and write BLE characteristics from devices and provide the results in an MQTT message.   
```

Comment:
Could you add maybe that this will be done when the BLEconnect is launched and not right after the command

Reply:
Will do